build_product

```diff
@@ -340,6 +340,6 @@ set -e
 rm -rf build/*
 cd build
 cmake .. "${CMAKE_OPTIONS[@]}"
-$build_command "-j${jobs}"
+$build_command "-j${jobs}" generate-ssg-rhel{6,7}-ds.xml 
```

Comment:
I guess this is not supposed to be here, :)

Reply:
that's true. It's the compulsive `-a` option to `git commit` hahah